### ISSUE TITLE
Accomodating new tables definition

### ISSUE THREAD

**Issue by aubertc** [chdemko/pandoc-numbering#18]
I believe the implementation of tables changed slightly with the release of [pandoc 2.10](https://github.com/jgm/pandoc/releases/tag/2.10), and it now makes pandoc-numbering crash.

For instance, a markdown file containing

```
Category    | Details |
--------------| ---------|
Category1 | Hourly | 
```

compiles just fine with 

```
pandoc test_pandoc-numbering.md - -o test_pandoc-numbering.html 
```

but crashes with 

```
pandoc test_pandoc-numbering.md --filter pandoc-numbering -o test_pandoc-numbering.html 
```

with the error message

```
Traceback (most recent call last):
  File "/usr/local/bin/pandoc-numbering", line 11, in <module>
    load_entry_point('pandoc-numbering==3.2.0.0', 'console_scripts', 'pandoc-numbering')()
  File "/usr/local/lib/python3.7/dist-packages/pandoc_numbering.py", line 1440, in main
    [numbering, referencing], prepare=prepare, doc=doc, finalize=finalize
  File "/usr/local/lib/python3.7/dist-packages/panflute/io.py", line 239, in run_filters
    doc = load(input_stream=input_stream)
  File "/usr/local/lib/python3.7/dist-packages/panflute/io.py", line 59, in load
    doc = json.load(input_stream, object_pairs_hook=from_json)
  File "/usr/lib/python3.7/json/__init__.py", line 296, in load
    parse_constant=parse_constant, object_pairs_hook=object_pairs_hook, **kw)
  File "/usr/lib/python3.7/json/__init__.py", line 361, in loads
    return cls(**kw).decode(s)
  File "/usr/lib/python3.7/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.7/json/decoder.py", line 353, in raw_decode
    obj, end = self.scan_once(s, idx)
  File "/usr/local/lib/python3.7/dist-packages/panflute/elements.py", line 1532, in from_json
    raise Exception('unknown tag: ' + tag)
Exception: unknown tag: Caption
Error running filter pandoc-numbering:
Filter returned error status 1
```
```

**Comment by chdemko** [comment]
It seems that the error comes from panflute which is used by pandoc-numbering

**Comment by aubertc** [comment]
Thanks for coming back to me.
I believe you are right, and that the error raised is around that line:

https://github.com/sergiocorreia/panflute/blob/master/panflute/elements.py#L1532

I don't have the expertise with panflute to open a minimal / interesting issue, as I don't know how to reproduce this issue without your filter…

**Comment by chdemko** [comment]
I think there is a link with this issue: https://github.com/sergiocorreia/panflute/issues/142

**Comment by sergiocorreia** [comment]
This should be a relatively easy fix, as it just involves adding a new element (Caption).

My main difficulty with adding 2.10 support though is that I can't test it easily as there is no support for writers/readers (so I don't know what exact JSON format to expect).

I can add the code, but without testing it's hard to know for sure if what I did is correct (e.g. if I set Caption as being a descendant of Inline types instead of Block types, Pandoc would fail to process panflute output.
